src/components/legacy/markdown/custom-components.tsx

```diff
@@ -131,19 +149,69 @@ const customSatellytesComponents = {
     return <SubTitle as={'h1'}>{props.children}</SubTitle>;
   },
   h2(props) {
-    return <SubTitle as={'h2'}>{props.children}</SubTitle>;
+    return (
+      <SubTitle as={'h2'}>
+        <a
```

Comment:
cool that works. What would you do regarding our fixed header though? Because the headlines slips under it when you click it. Do we need js for that or do we have other possibilities?

https://user-images.githubusercontent.com/1701755/158153162-06749665-639e-4fb6-b000-b158b771d6eb.mp4

Reply:
See me clicking on the headlines how it slips under the header.

Reply:
Thanks for the hint, I fixed it with `scroll-margin-top`